cedar-policy-core/src/evaluator.rs

```diff
@@ -208,6 +210,20 @@ impl<'e> Evaluator<'e> {
             },
             entities,
             extensions,
+            unknowns_mapper: Box::new(|_: &str| -> Option<Value> {None}),
+        }
+    }
+
+    // Constructs an Evaluator for a given unknowns mapper function.
+    pub(crate) fn with_unknowns_mapper(self, unknowns_mapper: Box<dyn Fn(&str) -> Option<Value> + 'e>) -> Self {
```

Comment:
Do you prefer to make a new, more generic constructor instead of this?

Reply:
Fixed with https://github.com/cedar-policy/cedar/pull/1466#issuecomment-2822587582